### PR TITLE
feat/HEYUI-131 Change 'In Development' status as 'Ready'

### PR DESCRIPTION
**Pull request by esraltintas**
### Description

Describe your changes (or add ticket description) and add relevant screenshots

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation change

### How do I test this

- Add steps to test
- in bullet point format
- preferably you can add link to the storybook build in the PR

### Did you remember to take care of the following?

- [ ] `npm i` – for new NPM dependencies.
- [ ] `npm run lint` - to check for linting issues
- [ ] `npm run test` - to run unit tests
- [ ] `npm run test:screenshots` - to run snapshot tests

### New Feature / Bug Fix

- [ ] Run unit tests to ensure all existing tests are still passing.
- [ ] Add new passing unit tests to cover the code introduced by your pr.

Thanks for contributing!

### CODE REVIEW

File: .storybook/blocks/Status/Status.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import styles from './Status.module.css';
 
 const STATUS = {
   0: 'Proposal',
-  1: 'In Development',
+  1: 'Ready',
```
[the remark this answers is not in the document]
Comment (esraltintas) [reply .storybook/blocks/Status/Status.tsx:7]: https://hey-car.slack.com/archives/C03TELS4P5G/p1661334143807779

**Comment by emrahsemizheycar** [comment]
It seems typography page was already in `review` status so after the changes it is showing up as `recommendation` can we also fix that ?

File: .storybook/blocks/Status/Status.module.css [hunk 2]
```diff
@@ -9,12 +9,7 @@
   background-color: var(--color-tarmac-grey-50);
 }
 
-.in_development {
-  color: var(--color-sunbeam-blue-900);
-  background-color: var(--color-sunbeam-blue-100);
-}
-
-.in_review {
+.ready {
```
Comment (derek-heycar) [inline .storybook/blocks/Status/Status.module.css:12]: Could you pls change the `ready` color to the green `--color-success-200` and the `Recommendation ` color to `--color-tertiary-200` or `--color-tertiary-300`just check what will be a better fit for it.

**Pull request merged**